cypress/integration/core-opensearch-dashboards/opensearch-dashboards/left-navigation/left_navigation.spec.js

```diff
@@ -75,36 +76,37 @@ if (isWorkspaceEnabled) {
     it('features are visible inside left navigation analytics for analytics use case', () => {
       validateWorkspaceNavMenu('all', () => {
         cy.contains('Visualize and report').should('exist');
+        cy.contains('Observability').should('exist');
+        cy.contains('Security Analytics').should('exist');
         cy.contains('Search').should('exist');
-        cy.contains('Manage workspace').should('exist');
+        cy.contains('Detect').should('exist');
       });
     });
 
     it('features are visible inside left navigation analytics for essentials use case', () => {
-      validateWorkspaceNavMenu('essentials', () => {
-        cy.contains('Manage workspace').should('exist');
-      });
+      validateWorkspaceNavMenu('essentials', () => {});
     });
 
     it('features are visible inside left navigation analytics for search use case', () => {
       validateWorkspaceNavMenu('search', () => {
         cy.contains('Visualize and report').should('exist');
-        cy.contains('Manage workspace').should('exist');
+        cy.contains('Compare search results ').should('exist');
```

Comment:
```suggestion
        cy.contains(/Compare search results/).should('exist');
```
I'd recommend to avoid exact match on the potential space.

Reply:
Done